pkg/prompb/remote.pb.go

```diff
@@ -68,7 +68,7 @@ type WriteRequest struct {
 	XXX_sizecache        int32        `json:"-"`
 }
 
-func (m *WriteRequest) Reset()         { *m = WriteRequest{Timeseries: m.Timeseries[:0]} }
```

Comment:
These are our changes. They shouldn't be backed out. Wonder if we should move these function to a different go file.

Reply:
Yeah, moving to a different file will be safer. I will keep our custom changes in `custom.pb.go` in the same dir so that it remains safe to know when changes happen.